queries.sql

```diff
@@ -1,3 +1,63 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT name FROM animals WHERE name ILIKE '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT name FROM animals WHERE date_of_birth BETWEEN DATE '2016-01-01' AND '2019-01-01';
+
+-- List the name of all animals that are neutered and have less than 3 escape attempts.
+SELECT name FROM animals WHERE neutered = 'true' AND escape_attempts < 3;
+
+-- List the date of birth of all animals named either "Agumon" or "Pikachu".
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+
+-- List name and escape attempts of animals that weigh more than 10.5kg
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+
+-- Find all animals that are neutered.
+SELECT * FROM animals WHERE neutered = 'true';
+
+-- Find all animals not named Gabumon.
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
+
+-- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
+SELECT * FROM animals WHERE weight_kg BETWEEN '10.4' AND '17.3';
+
+-- DAY - 2
+
```

Comment:
- [ ] Insiede a transaction,`UPDATE` the animals table by setting the species column to unspecified, then roll back the change here as mentioned in the project requirements.

- [ ] Inside a transaction,
* `UPDATE` the animals table by setting the species column to digimon for all animals that have a name ending in mon.
* `UPDATE` the animals table by setting the species column to pokemon for all animals that don't have species already set.
* Commit the transaction.

- [ ] Inside a transaction delete all records in the animals table, then roll back the transaction as menioned in the project requirements.

Reply:
- [x] Insiede a transaction,UPDATE the animals table by setting the species column to unspecified, then roll back the change here as mentioned in the project requirements.

![set species to unspecified and rollback](https://github.com/Paulinelali/vet-clinic-project/assets/88007635/d319168d-a061-4b27-b966-ada4859231bc)

- [x]  Inside a transaction,

UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.
UPDATE the animals table by setting the species column to pokemon for all animals that don't have species already set

![update species to digimon or pokemon](https://github.com/Paulinelali/vet-clinic-project/assets/88007635/f827fa13-112e-4096-8cfd-e70eff38a99d)

- [x] Inside a transaction delete all records in the animals table, then roll back the transaction as menioned in the project requirements.

![delete all data in animals table and rollback](https://github.com/Paulinelali/vet-clinic-project/assets/88007635/10ac25c5-e043-419f-ae60-455c923ea3b1)

Reply:
you did well by correcting some of the changes by the previous reviewer, however there are some changes that wasn't implemented :

- Inside a transaction,UPDATE the animals table by setting the species column to unspecified, then roll back the change here as mentioned in the project requirements.


-  Inside a transaction,

UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.
UPDATE the animals table by setting the species column to pokemon for all animals that don't have species already set.
Commit the transaction.

-  Inside a transaction delete all records in the animals table, then roll back the transaction as menioned in the project requirements.


-  Inside a transaction, UPDATE the animal's table by setting the species column to unspecified, then roll back the change here as mentioned in the project requirements.

- Inside a transaction,

- UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.

- UPDATE the animals table by setting the species column to Pokemon for all animals that don't have species already set.

Commit the transaction.
 Inside a transaction delete all records in the animal's table, then roll back the transaction as mentioned in the project requirements.

-  Inside a transaction, UPDATE the animal's table by setting the species column to unspecified, then roll back the change here as mentioned in the project requirement

- UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.
UPDATE the animals table by setting the species column to Pokemon for all animals that don't have species already set

-  Inside a transaction delete all records in the animal's table, then roll back the transaction as menioned in the project requirements.